ui/server.py

```diff
@@ -23,6 +25,11 @@
             template_folder=os.path.join(APP_ROOT, './templates'))
 app.config['UPLOAD_FOLDER'] = os.path.join(APP_ROOT, './backend')
 app.config['DEBUG'] = True  # Remove this line in production
+app.config['SECRET_KEY'] = os.getenv('SECRET_KEY')
+
+# HTTPS = True if both a certificate and private key exist, False otherwise.
+HTTPS = (os.getenv('SSL_certificate') !=
+         '' and os.getenv('SSL_private_key') != '')
```

Comment:
In python, parentheses are not needed in if-clauses (unless you need more than 1 line) - pep8
Moreover, `if os.getenv('SSL_certificate') and os.getenv('SSL_private_key'):` should be sufficient